src/main/java/org/folio/circulation/domain/LoanRepresentation.java

```diff
@@ -34,6 +35,12 @@ public JsonObject extendedLoan(Loan loan) {
       extendedRepresentation.remove(LoanProperties.BORROWER);
     }
 
+    if (loan.getLoanPolicy() != null) {
+      additionalLoanPolicyProperties(extendedRepresentation, loan.getLoanPolicy());
+    }else{
```

Comment:
Please add spaces between the braces and the else statement
```suggestion
    } else {
```

Reply:
Done